src/utils/attribute.mapper.util.js

```diff
@@ -51,34 +52,35 @@ export default async function mapAttributesToTicket(req, res, next) {
  * @param {object} culr The CULR reponse object
  * @param {object} attributes The attributes object defined in Smaug
  * @param {object} user data returned by the idp
- * @param clientId
- * @param accessToken
+ * @param {string} clientId
+ * @param {string} accessToken
  * @see ATTRIBUTES
  * @returns {Promise<{}>}
  */
-export async function mapCulrResponse(
-  culr,
-  attributes,
-  user,
-  clientId,
-  accessToken
-) {
-  let mapped = {};
+export async function mapCulrResponse(culr, attributes, user, clientId, accessToken) {
+  const agencies = [];
+  const attributeKeys = Object.keys(attributes);
+  const mapped = {
+    serviceStatus: {
+      borchk: culr.errorBorchk ? 'error' : 'ok',
+      culr: culr.errorCulr ? 'error' : 'ok'
+    }};
   let cpr = user.cpr || null;
-  let agencies = [];
-  let fromCpr = {};
   let dbcidpAuthorize;
   let foundLoginAgency = false;
   let clientInfo = {};
 
   log.debug('mapCulrResponse', culr);
 
+  const branchInfo = user.agency ? await getAgency(user.agency) : {};
```

Comment:
vil branchInfo altid! udlevere en logbinAgencyId ?

Ellers kan der jo ske en fejl i line 83.

Reply:
Jeps, det sørger den underliggende kode for